src/main/java/az/gdg/msteam/service/impl/MemberServiceImpl.java

```diff
@@ -0,0 +1,130 @@
+package az.gdg.msteam.service.impl;
+
+import az.gdg.msteam.exception.MemberNotFoundException;
+import az.gdg.msteam.mapper.MemberMapper;
+import az.gdg.msteam.model.MemberResponse;
+import az.gdg.msteam.model.dto.MemberDto;
+import az.gdg.msteam.model.entity.Member;
+import az.gdg.msteam.repository.MemberRepository;
+import az.gdg.msteam.service.MemberService;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class MemberServiceImpl implements MemberService {
+
+    MemberRepository memberRepository;
+    private boolean isValid;
+
+    public MemberServiceImpl(MemberRepository memberRepository) {
+        this.memberRepository = memberRepository;
+    }
+
+    @Override
+    public List<MemberDto> getAllMembers() {
```

Comment:
why? Just retrieve the list from repository and throw exception if nothing is found. Do not overcomplicate. Review my code to see how it is done.